snipeit/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: snipeit
-version: 3.3.1
-appVersion: 5.3.6
+version: 3.4.0
+appVersion: 6.0.14
```

Comment:
The SnipeIT Images are prepended with "v". So when you set the image tag to .Chart.AppVersion with a version defined without "v", it will not be found on dockerhub.
Please use v6.0.14.

Reply:
Actually, I was waiting for tests to pass to test specifically this, but it... passed as it is only linting! Fixing.

Reply:
Fixed.